sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -232,25 +237,39 @@ object Webpack {
       .getOrElse(generatedWebpackConfigFile.file)
 
     val args = extraArgs ++: Seq("--config", configFile.absolutePath)
-    Webpack.run(args: _*)(generatedWebpackConfigFile.targetDir, log)
+    val stats = Webpack.run(args: _*)(generatedWebpackConfigFile.targetDir, log)
+    stats.foreach(_.print(log))
 
-    val library = generatedWebpackConfigFile.asLibrary
+    val library = generatedWebpackConfigFile.asLibrary(stats)
     assert(library.file.exists, "Webpack failed to create library file")
     library
   }
 
+  private def jsonOutput(logger: Logger)(in: InputStream): Option[WebpackStats] = {
+
+    val parsed = Json.parse(in)
+    parsed.validate[WebpackStats] match {
+      case JsError(e) =>
+        // In case of error print the result and return None. it will be ignored upstream
+        e.foreach {
+          case (p, v) => logger.error(s"$p: ${v.mkString(",")}")
```

Comment:
We should probably also print a first line saying that we were unable to decode the webpack output